.github/workflows/ci.yml

```
@@ -0,0 +1,150 @@
+name: CI
+on:
+  pull_request:
+    branches: [development]
+    paths-ignore:
+      - '*.md'
+  push:
+    branches: [development]
+    paths-ignore:
+      - '*.md'
+concurrency:
+  group: ci-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+jobs:
+  test-build-typescript:
+    name: Build Typescript
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup node and npm registry
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+          registry-url: 'https://registry.npmjs.org/'
+          cache: 'yarn'
+      - name: Install node_modules
```

```suggestion

      - name: Install node_modules
```

I think with the suggestions until here, the pattern I am anticipating should be clear. Let me know if there are questions, or if you want to discuss the pattern in general (e.g. because you do not agree with it).